### PR TITLE
Transactions history modal

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/1211

also:

https://github.com/AugurProject/augur/issues/1572 (finalize was dropped until v2, we don't store those txhashes in AN yet)

and:

https://github.com/AugurProject/augur/issues/1341

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** and **fixes 1** when merging 78762257172f068f28c0a69aaf00166915ee6f6f into 404dd19668a0e7ddaa8f7699ffe0c6b0797b357b - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-771f625d3c06993c27dadb5d6ea51b01fe6713d8)

**new alerts:**

* 1 for Potentially inconsistent state update

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by AugurIntegration** [comment]
This pull request **fixes 1 alert** when merging fc9d69383a5aecb7a6e16964ac2882e2636804e3 into e5c0c2ad95df5f77d05318e52b28d2acbddc3289 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-47e1b3445f8df1627394a99ebe0f521a4003ec19)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by AugurIntegration** [comment]
This pull request **fixes 1 alert** when merging 188dcd907bad4ccf5b526aca4deacceb4c80998d into e5c0c2ad95df5f77d05318e52b28d2acbddc3289 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-53eb08dc34b3c35d06acb9fa2e30eb7eb0e3bed3)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: tsconfig.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     /* Basic Options */
     "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2015"],                             /* Specify library files to be included in the compilation. */
+    "lib": ["es2015", "dom"],                             /* Specify library files to be included in the compilation. */
```
Comment (andrewdamelio) [inline tsconfig.json]: Did this fix our TS issues?

**Comment by AugurIntegration** [comment]
This pull request **fixes 1 alert** when merging a1541ce8d337bbaa0665dbb4c7a242818fa9548e into bad9fe361379df3edde7029f9d6a2de06c0b9913 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-3dab97e26a23a2cf303b6c4f9e2ace7b451804d6)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**